### PR TITLE
support policy local running

### PR DESCRIPTION
**Pull request by zagronitay**

### CODE REVIEW

File: pkg/client/client.go [hunk 1]
```diff
@@ -88,8 +88,11 @@ func (p ProviderFetchSummary) HasErrors() bool {
 
 // PolicyRunRequest is the request used to run a policy.
 type PolicyRunRequest struct {
-	// Args are the given arguments from the policy run command.
-	Args []string
+	// Remote is the given remote policy from the run command.
+	Remote string
+
+	// Local is the given remote policy from the run command.
```
Comment (zagronitay) [inline pkg/client/client.go:94]: Will change the comment

**Review by michelvocks — APPROVED** [review]
LGTM

**Review by spangenberg — APPROVED** [review]
LGTM

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Something feels a bit weird here with the flags. Why two flags are necessary? what happens if someone does `--remote` and `--local`. I feel like one should be the default then. 

let's do remote by default? unless `--local` is specified?

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**